test/common/upstream/health_checker_impl_test.cc

```diff
@@ -3247,6 +3247,8 @@ class GrpcHealthCheckerImplTestBase {
                     headers.Scheme()->value().getStringView());
           EXPECT_NE(nullptr, headers.Method());
           EXPECT_EQ(expected_host, headers.Host()->value().getStringView());
+          EXPECT_GT(Envoy::Grpc::Common::getGrpcTimeout(headers).count(),
```

Comment:
nit: can you just just EXPECT_EQ the exact value?

Reply:
sure, but why? wouldn't that just add brittleness to the test (e.g. if some other change sets it to something else than 1000ms for whatever reason)? maybe validating presence and checking that it's non-zero is more correct? :)

Reply:
Because I don't see how the test timeout will change and this will verify we aren't somehow changing the value to something else. I see no reason not to do an exact comparison here. Thank you!

/wait

Reply:
fair enough, fixed